app/serotracker_sqlalchemy/db_model_config.py

```diff
@@ -27,19 +27,22 @@
     ]
 }
 
-db_tables = ['airtable_source',
+db_tables = ['dashbord_source',
+             'research_source',
              'city',
              'city_bridge',
              'state',
              'state_bridge',
              'test_manufacturer',
-             'test_manufacturer_bridge']
+             'test_manufacturer_bridge',
+             'antibody_target',
+             'antibody_target_bridge']
 
-dashboard_source_cols = ['source_name', 'source_type', 'study_name', 'denominator_value',
+dashboard_source_cols = ['source_id', 'source_name', 'source_type', 'study_name', 'denominator_value',
                          'overall_risk_of_bias', 'serum_pos_prevalence', 'isotype_igm', 'isotype_iga',
                          'isotype_igg', 'sex', 'age', 'sampling_start_date', 'sampling_end_date', 'estimate_grade',
                          'isotype_comb', 'academic_primary_estimate', 'dashboard_primary_estimate', 'pop_adj',
-                         'test_adj', 'specimen_type', 'test_type', 'population_group', 'url']
+                         'test_adj', 'specimen_type', 'test_type', 'population_group', 'url', 'gbd_region']
 
 research_source_cols = ['case_population', 'deaths_population', 'age_max', 'age_min', 'age_variation',
```

Comment:
Seems that research_source_cols is defined in this file and in airtable_fields_config - can we just have it in one place so there is less for us to keep track of and maintain?

Reply:
Actually there might be some subtle differences between the airtable_fields_config and db_model_config versions. I'm gonna clean things, close up this PR, and re request review later